package.json

```diff
@@ -53,21 +36,21 @@
   "main": "./lib/index.js",
   "oclif": {
     "commands": "./lib/commands",
+    "topicSeparator": " ",
```

Comment:
I don't think we want to add this. It won't matter when we include it in `sf` since plugins inherit the topic separator

Reply:
Whoops, yeah you're right I added it when I was playing around with things.